released/usecases/services_apps/sap_build_code/step1/main.tf

```diff
@@ -94,7 +101,7 @@ resource "btp_subaccount_entitlement" "destination" {
   service_name  = "destination"
```

Comment:
@rui1610 I'm assuming we can't create the cicd destination without the destination service instance? I couldn't find the respective terraform BTP resource.